src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_4001_5000.cpp

```diff
@@ -6030,6 +6030,39 @@ GUI_TEST_CLASS_DEFINITION(test_4983) {
     GTUtilsLog::check(os, l);
 }
 
+GUI_TEST_CLASS_DEFINITION(test_4985) {
+    // Create file _common_data/scenarios/sandbox/A.fa with text "A"
+    // Open this file
+    // Delete this file from hard disk
+    // "File Modification Detected" dialog appears. Click "No"
+    // On Start Page open recent file A.fa
+    //      Expected: error message box with text "File doesn't exist: _common_data\scenarios\sandbox\a.fa" appears
+
+    QFile file(testDir + "_common_data/scenarios/sandbox/A.fa");
+    file.open(QIODevice::WriteOnly | QIODevice::Text);
+    QTextStream out(&file);
+    out << 'A';
+    out.flush();
+    CHECK_SET_ERR(file.size() == 1, "Error writing file " + file.fileName())
+
+    GTFileDialog::openFile(os, file.fileName());
+    GTUtilsTaskTreeView::waitTaskFinished(os);
```

Comment:
Check that something is changed: an object view is opened or project tree has the document.